requirements.txt

```diff
@@ -12,3 +12,4 @@ pytest==7.3.1
 pytest-cov==4.1.0
 pytest-xdist==3.5.0
 setuptools==59.6.0
+open3d==0.18.0 
```

Comment:
Can you insert this where it would be alphabetically? Helps make sure there are no duplicates.